src/main/java/com/minecolonies/coremod/commands/ActionArgumentType.java

```diff
@@ -232,7 +236,8 @@ public int allowedSpaceCount()
     }
 
     @NotNull
-    private List<String> getCitizenTabCompletions(@NotNull final ActionMenuState actionMenuState, final String potentialArgumentValue)
+    private List<String> getCitizenTabCompletions(@NotNull final MineColonyDataProvider mineColonyDataProvider,
+            @NotNull final ActionMenuState actionMenuState, final String potentialArgumentValue)
     {
         // TODO: see if we can figure out what citizen we are looking at as the default tab completion.
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)